### PR TITLE
Testing annotations

### PR DESCRIPTION
**Pull request by Mogztter**

### CODE REVIEW

File: articles/modules/ROOT/pages/invalid.adoc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+= Test linter
```
Comment (Mogztter) [inline articles/modules/ROOT/pages/invalid.adoc:1]: Remove this invalid file before merging

**Review by jexp — APPROVED** [review]
@Mogztter can you document the linting a bit more in detail?

**Comment by Mogztter** [comment]
@jexp I've updated the README and made some adjustments, let me know if it makes sense now.
Since I've created this pull request from my fork, I don't have write permissions to create annotations. Having said that, I did a few tests on my fork and it's working fine.

Once this is merged, it would be great to rebase existing pull requests to see if the annotations are working as expected.
We would be able to add new rules.

**Comment by jexp** [comment]
Needs rebase.

**Comment by jexp** [comment]
@Mogztter can you rebase so I can merge this?

**Comment by Mogztter** [comment]
@jexp done but https://github.com/neo4j-documentation/knowledge-base/pull/61 needs to go first. The UI bundle was removed from https://github.com/neo4j-documentation/docs-refresh.
The source code of the UI bundle now lives at https://github.com/neo4j-documentation/docs-ui and the `ui-bundle.zip` file is published to S3.

**Pull request merged**